iocage_lib/ioc_json.py

```diff
@@ -2826,3 +2825,23 @@ def json_migrate_uuid_to_tag(self, uuid, tag, state, conf):
             conf["host_hostname"] = tag
 
         return (conf, False, True)
+
+    @staticmethod
+    def validate_ip4_addr(ip):
+        # PTP configurations have two addresses.  Otherwise there will be 1.
+        parts = ip.split(" ")
+        if len(parts) > 2:
+            raise ValueError("Unrecognized IP4 address format")
+        if len(parts) == 2:
+            ipaddress.IPv4Address(parts[1])
+        ipaddress.IPv4Network(parts[0], strict=False)
```

Comment:
why is only one address validated with strict false?

Reply:
I don't remember.  It's been too long since I did the original work.